tests/softlearning/environments/adapters/robosuite_adapter_test.py

```diff
@@ -0,0 +1,105 @@
+import unittest
+
+from .softlearning_env_test import AdapterTestClass
+from softlearning.environments.adapters.robosuite_adapter import (
+    RobosuiteAdapter)
+
+class TestRobosuiteAdapter(unittest.TestCase, AdapterTestClass):
+    # TODO(hartikainen): This is a terrible way of testing the envs.
+    # All the envs should be tested independently.
+
+    def create_adapter(self, domain='Sawyer', task='Lift', *args, **kwargs):
+        return RobosuiteAdapter(
+            domain,
+            task,
+            *args,
+            **kwargs,
+            has_renderer=False,
+            has_offscreen_renderer=False,
+            use_camera_obs=False)
+
+    def test_environments(self):
+        # Make sure that all the environments are creatable
+        TEST_ENVIRONMENTS = [('Sawyer', 'Lift')]
+        def verify_reset_and_step(domain, task):
+            env = RobosuiteAdapter(
+                domain=domain,
+                task=task,
+                has_renderer=False,
+                has_offscreen_renderer=False,
+                use_camera_obs=False)
+            env.reset()
+            env.step(env.action_space.sample())
+
+        for domain, task in TEST_ENVIRONMENTS:
+            print("testing: ", domain, task)
+            verify_reset_and_step(domain, task)
+
+    def test_copy_environments(self):
+        domain, task = 'Sawyer', 'Lift'
+        env_kwargs = {
+            "gripper_type": "TwoFingerGripper",
+            "table_full_size": (0.8, 0.8, 0.8)
+        }
+        env1 = self.create_adapter(domain=domain, task=task, **env_kwargs)
+        env1.reset()
+        env2 = env1.copy()
+
+        self.assertEqual(env1.observation_keys, env2.observation_keys)
+        for key, value in env_kwargs.items():
+            self.assertEqual(getattr(env1.unwrapped, key), value)
+            self.assertEqual(getattr(env2.unwrapped, key), value)
+
+        domain, task = 'Sawyer', 'Lift'
+        robosuite_adapter_kwargs = {
+            'observation_keys': ('joint_pos', 'joint_vel')
```

Comment:
This used to pass with incorrect `observation_keys`. Can you add a test that makes sure that incorrect observation keys fail?

Reply:
Yep, implemented!